.github/workflows/build-status-check.yml

```diff
@@ -15,8 +15,10 @@ on:
 jobs:
   build_status_check:
     name: ${{ vars.PROJECT_NAME }} Build Status Check
-    uses: KinsonDigital/Infrastructure/.github/workflows/build-csharp-project.yml@v13.0.0
+    uses: KinsonDigital/Infrastructure/.github/workflows/build-csharp-project.yml@v13.1.0
     with:
       project-name: "${{ vars.PROJECT_NAME }}"
       build-config: Debug
       net-sdk-version: "${{ vars.NET_SDK_VERSION }}"
+      checkout-repository: ${{ github.repository }}
+      checkout-ref: ${{ github.head_ref }}
```

Comment:
1. These need to be updated to match what we got in Velaptor
> **Note**
> This is my bad. I should have referenced Velaptor to make sure we were doing it correctly in the first place.
3. Also, make sure that there is a single empty line at the end of the file.

**Velaptor Implementation:**
<img width="475" alt="image" src="https://github.com/KinsonDigital/CASL/assets/105235056/39d3f067-8ad1-4f4c-8e5d-4bff65caa76c">